.github/workflows/ci-conda-workflow.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             test-data: submodule
             matplotlib-version: 3
             bokeh-version: 3
-            xspec-version: 12.14.0i
+            xspec_version: 12.14.0i
```

Comment:
There's no reason for changing this (note that `bokeh-version` and `matplotlib-version` above have not been changed).

Reply:
too much replace all. Will fix.